joomla/joomla-cms
# [4] Empty State for System-> Discover

### Summary of Changes

Empty state for when no extensions are discovered.

MOST OF THE TIME you will expect to see the Empty State. Its rare to see extensions needing discovering. This is the opposite of most components in Joomla admin. 


### Testing Instructions

Apply PR

Load System -> discover,

See Empty State 

click discover... or the button in the middle

Note new message. Nothing found. 

upload a plugin/extension manually into its right place using FTP (or copy and paste on your local machine) 

Load System -> discover, click discover... 

See you see the old screen where you can install the extension

### Actual result BEFORE applying this Pull Request

<img width="1600" alt="Screenshot 2021-04-28 at 23 46 50" src="https://user-images.githubusercontent.com/400092/116481896-05b39b80-a87c-11eb-99dd-88b6395a89c6.png">


### Expected result AFTER applying this Pull Request

<img width="1329" alt="Screenshot 2021-04-28 at 23 43 50" src="https://user-images.githubusercontent.com/400092/116481837-e6b50980-a87b-11eb-8117-f48042a29ca8.png">

When you click either discover button you get a new message, or redirected to the list of discovered extensions to install (as normal) 

<img width="1341" alt="Screenshot 2021-04-28 at 23 43 56" src="https://user-images.githubusercontent.com/400092/116481838-e87ecd00-a87b-11eb-9e2b-7ece2371a4be.png">


### Documentation Changes Required

none
